packages/grafana-ui/src/components/GraphNG/GraphConfigBuilder.tsx

```diff
@@ -0,0 +1,143 @@
+import tinycolor from 'tinycolor2';
```

Comment:
I think this builder should be part of the Plot component (the uPlot abstraction), not the GraphNG component. 

This builder is about mapping Grafana high-level prop concepts & types to internal uPlot config.